src/Components/Publishing/Header/Layouts/Components/__tests__/ClassicPromotedContent.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from "@artsy/palette"
+import { ClassicArticlePromotedContent } from "Components/Publishing/Fixtures/Articles"
+import { mount } from "enzyme"
+import { cloneDeep } from "lodash"
+import React from "react"
+import { ClassicPromotedContent } from "../ClassicPromotedContent"
+
+describe("FeatureBasicHeader", () => {
+  let props
+  const getWrapper = (passedProps = props) => {
+    return mount(<ClassicPromotedContent {...passedProps} />)
+  }
+
+  beforeEach(() => {
+    props = {
+      article: cloneDeep(ClassicArticlePromotedContent),
+    }
+  })
+
+  it("Renders expected content for partner galleries", () => {
+    delete props.article.sale
+    props.article.partner = {
+      default_profile_id: "contessa-gallery",
+      name: "Contessa Gallery",
+      type: "Gallery",
+      profile: {
+        id: "contessa-gallery",
+        href: "/contessa-gallery",
+        image: {
+          cropped: {
+            url:
+              "https://d196wkiy8qx2u5.cloudfront.net?resize_to=fill&width=250&height=165&quality=80&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FOyZ08tRGQ1k-ue1nk06vlQ%2Fsquare.jpg",
+          },
+        },
+      },
+    }
+    const component = getWrapper()
+    expect(component.text()).toMatch("Promoted Content")
+    expect(component.text()).toMatch("Contessa Gallery")
+    expect(component.find(Button).text()).toMatch("Explore Gallery")
+    expect(component.html()).toMatch("square.jpg")
+  })
+
+  it("Renders expected content for auctions", () => {
+    const component = getWrapper()
```

Comment:
I'm not requesting changes, but one technique that can be useful for reading tests is to set up the data that makes this test unique inside of this test. We do it in the previous test - `props.article.partner = ....`. 

In _this_ test, my assumption is that there is a sale object in the default fixture, but I have to go read that fixture file to make sure. If we also set `props.article.sale` at the top of this test, even if it's duplication, it gives me a good sense of what data is needed for this test to pass.